docs/src/modules/spring/pages/workflow-entities.adoc

```diff
@@ -107,4 +107,4 @@ IMPORTANT: For simplicity purposes, we are returning the internal state directly
 
 //TODO update the `here` link after merge, otherwise the docs validation complain
 
-A full transfer workflow source code is available here. Follow the `Readme` file to run and test it.
+A full transfer workflow source code is available {sample-url}[here]. Follow the `README` file to run and test it.
```

Comment:
I think usually we force the link to open on a new tab if it's an external link. I find it useful.

```suggestion
A full transfer workflow source code is available {sample-url}[here {tab-icon}, window="new"]. Follow the `README` file to run and test it.
```

Reply:
I felt the responsability to fix it after my suggestion broke CI :D

Reply:
thanks :)